docs/installation.md

```diff
@@ -33,14 +33,6 @@ platforms.
 > hipSYCL is currently available on Linux and has experimental/partial support
 > for OSX and Windows.
 
-### ComputeCpp
```

Comment:
We also still have this in our main README

Reply:
Removed.